stored_responses/stored_responses_test.go

```diff
@@ -590,6 +590,216 @@ func TestProcessStoredAuctionAndBidResponses(t *testing.T) {
 
 }
 
+func TestProcessStoredResponsesNotFoundResponse(t *testing.T) {
```

Comment:
I think it'd be nice to have one test case where we expect no errors with valid stored response information. What do you think?

Reply:
Yes, agree. It exists already: `TestProcessStoredAuctionAndBidResponses`. Before this change we didn't throw an error and this functionality is covered. With this change valid cases are still valid and this specific test was added to cover error cases only.